client/transaction-pool/graph/src/pool.rs

```diff
@@ -105,12 +105,12 @@ impl Default for Options {
 	fn default() -> Self {
 		Options {
 			ready: base::Limit {
-				count: 512,
-				total_bytes: 10 * 1024 * 1024,
+				count: 8192,
+				total_bytes: 20 * 1024 * 1024,
 			},
 			future: base::Limit {
-				count: 128,
-				total_bytes: 1 * 1024 * 1024,
+				count: 2048,
```

Comment:
I don't think that `future` pool should be that big. It should really be empty most of the time. I'd suggest leaving it as low as `512 / 1 MB`